src/js/rx/components/Breadcrumbs.jsx

```diff
@@ -30,7 +30,7 @@ class Breadcrumbs extends React.Component {
     }
 
     return (<div className="rx-breadcrumbs">
-      {crumbs.reduce((content, e) => { return [...content, ' › ', e]; }, crumbs[0]).slice(1)}
+      {crumbs.reduce((content, e) => { return [...content, ' › ', e]; }, []).slice(1)}
```

Comment:
Wouldn't the initial value be `[crumbs[0]]`? When it was `crumbs[0]` by itself, the `...` operator probably didn't work correctly on it.